charts/civil-general-applications/values.yaml

```diff
@@ -14,6 +14,7 @@ camunda-bpm:
   enabled: false
 
```

Comment:
Hi, just in this file it looks like a default value for civil-service.enabled is missing

Reply:
sure just added